### PR TITLE
[FEATURE] Localize Accessibility Strings

### PR DESCRIPTION
**Pull request by palatter**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingViewComponent/ControlBarViewModel.swift [hunk 1]
```diff
@@ -78,7 +91,8 @@ class ControlBarViewModel: ObservableObject {
                 self.logger.debug("Hangup button tapped")
                 self.endCallButtonTapped()
         }
-        self.hangUpButtonViewModel.accessibilityLabel = "End call"
+        hangUpButtonViewModel.accessibilityLabel = self.localizationProvider.getLocalizedString(
```
Comment (JoshuaLai) [inline AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingViewComponent/ControlBarViewModel.swift:94]: Just a small call out here: The previous accessibility label was saying "End call" and now you are saying "Leave call" I assume this is intended

**Review by JoshuaLai — APPROVED** [review]

File: AzureCommunicationUI/AzureCommunicationUITests/Presentation/Calling/ParticipantGridsViewModelTests.swift [hunk 2]
```diff
@@ -298,7 +298,7 @@ class ParticipantGridViewModelTests: XCTestCase {
         let state = makeRemoteParticipantState(count: 2)
         let callingState = CallingState(status: .connected)
         let accessibilityProvider = AccessibilityProviderMocking()
-        let expectedAnnouncement = "2 participants joined the meeting"
+        let expectedAnnouncement = "AzureCommunicationUI.CallingView.MutiplePeopleJoined"
```
Comment (vhuseinova-msft) [inline AzureCommunicationUI/AzureCommunicationUITests/Presentation/Calling/ParticipantGridsViewModelTests.swift:301]: Could you switch to LocalizationProvider usage so we can check that the used parameter is also correct?

**Review by vhuseinova-msft — APPROVED** [review]
Minor comment, otherwise lgtm
Tested.

File: AzureCommunicationUI/AzureCommunicationUITests/Presentation/Calling/ParticipantGridsViewModelTests.swift (hunk 2 above)
Comment (jimchou-dev) [reply AzureCommunicationUI/AzureCommunicationUITests/Presentation/Calling/ParticipantGridsViewModelTests.swift:301]: You can use LocalizableKey enum with rawValue

Comment (vhuseinova-msft) [reply AzureCommunicationUI/AzureCommunicationUITests/Presentation/Calling/ParticipantGridsViewModelTests.swift:301]: But in this case ParticipantGridViewModel will still return a key as it uses LocalizationProviderMocking

**Pull request merged**